src/controller/discoverymgr/mnedc/tunmgr/tun.go

```diff
@@ -36,7 +37,7 @@ var (
 )
 
 func init() {
-	// Do nothing because there is no need to initialize anything
```

Comment:
Do you have a reason to get rid of this line?

Reply:
No, I will revert this.